src/OpenTelemetry.AutoInstrumentation/Instrumentations/StackExchangeRedis/StackExchangeRedisInitializer.cs

```diff
@@ -23,18 +23,18 @@ namespace OpenTelemetry.AutoInstrumentation.Instrumentations.StackExchangeRedis;
 
 internal static class StackExchangeRedisInitializer
 {
-    public static void Initialize(object connection)
+    public static void Initialize(object? connection)
```

Comment:
`connection` seems a candidate for refactoring, no point to call with `null` value.

Reply:
Two conditional calls in place where we use it, or accept null here, I think that we can keep nullable version here.

Reply:
16ce6223c259f140db177d5c08e2ca521f8366d2